RELEASES.md

```diff
@@ -0,0 +1,60 @@
+Planing for new release starts with relabeling all candidates that were not completed in the current release - in example [1.11-candidates](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A1.11-candidate) and labeling all the new for the upcoming release.
+
+In parallel we start a [discussion](https://github.com/vmware-tanzu/velero/discussions?discussions_q=label%3ASurvey) to collect new ideas, proposals, needs and new contributors.
+
+# Versioning and Release
+This document describes the versioning and release process of Velero. This document is a living document, contents will be updated according to each releases.
+
+## Releases
+Velero releases will be versioned using dotted triples, similar to [Semantic Version](http://semver.org/). For this specific document, we will refer to the respective components of this triple as `<major>.<minor>.<patch>`. The version number may have additional information, such as "-rc1,-rc2,-rc3" to mark release candidate builds for earlier access. Such releases will be considered as "pre-releases".
+
+### Major and Minor Releases
+Major and minor releases of Velero will be branched from `main` when the release reaches to `RC(release candidate)` state. The branch format should follow `release-<major>.<minor>.0`. For example, once the release `v1.0.0` reaches to RC, a branch will be created with the format `release-1.0.0`. When the release reaches to `GA(General Available)` state, The tag with format `v<major>.<minor>.<patch>` and should be made with command `git tag -s v<major>.<minor>.<patch>`. The release cadence is around 3 months, might be adjusted based on open source event, but will communicate it clearly.
+
+### Patch releases
+Patch releases are based on the major/minor release branch, the release cadence for patch release of recent minor release is one month to solve critical community and security issues. The cadence for patch release of recent minus two minor releases are on-demand driven based on the severity of the issue to be fixed.
+
+### Pre-releases
+`Pre-releases:mainly the different RC builds` will be compiled from their corresponding branches. Please note they are done to assist in the stabilization process, no guarantees are provided.
+
+### Minor Release Support Matrix
+| Version | Supported          |
+| ------- | ------------------ |
+| Velero v1.10.x   | :white_check_mark: |
+| Velero v1.9.x   | :white_check_mark: |
+| Velero v1.8.x   | :white_check_mark: |
+
+### Upgrade path and support policy
+The upgrade path for Velero is (1) 2.2.x patch releases are always compatible with its major and minor version. For example, previous released 2.2.x can be upgraded to most recent 2.2.3 release. (2) Velero only supports two previous minor releases to upgrade to current minor release. For example, 2.3.0 will only support 2.1.0 and 2.2.0 to upgrade from, 2.0.0 to 2.3.0 is not supported. One should upgrade to 2.2.0 first, then to 2.3.0.
```

Comment:
I would jump lines here to be easier to read. ex.:

The upgrade rule for Velero is 
- Patch releases are always compatible with its major and minor version. 
  For example: previous released 2.2.0, 2.2.1 and 2.2.2 can be upgraded to most recent 2.2.3 release.
- Only the two previous minor releases can be to upgraded to current minor release.
  For example: 2.1.0 and 2.2.0 can be upgraded to 2.3.0, but 2.0.0 can not be upgraded to 2.3.0. One should upgrade to 2.2.0 first, then to 2.3.0.